packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -43,10 +43,14 @@ import {
     runSweepKey,
     defaultInactiveTimeoutMs,
     gcTestModeKey,
+    disableSweepLogKey,
 } from "../garbageCollection";
 import { dataStoreAttributesBlobName, GCVersion, IContainerRuntimeMetadata, IGCMetadata } from "../summaryFormat";
 import { IGCRuntimeOptions } from "../containerRuntime";
 
+/** @see - sweepReadyUsageDetectionSetting */
+const SweepReadyUsageDetectionKey = "Fluid.GarbageCollection.Dogfood.SweepReadyUsageDetection";
```

Comment:
You can export this from `garbageCollection.ts` and not from `index.ts`. That way it can be used in the `containr-runtime` folder without this being publicly exported from the package.

Reply:
I wanted it hidden from the code even in garbageCollection.ts since it has special semantics.  I don't mind duplicating magic strings into test files because the test fails if they don't match.